client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetailsSchema.ts

```diff
@@ -134,7 +134,7 @@ const ClinicalDetailsSchema = (validationDate: Date) => {
         [ClinicalDetailsFields.HOSPITAL]: yup.string().nullable(),
         [ClinicalDetailsFields.HOSPITALIZATION_START_DATE]: wasHospitilizedStartDateSchema,
         [ClinicalDetailsFields.HOSPITALIZATION_END_DATE]: wasHospitilizedEndDateSchema,
-        [ClinicalDetailsFields.IS_PREGNANT]: yup.boolean().nullable().required(),
+        [ClinicalDetailsFields.IS_PREGNANT]: yup.boolean().nullable().required(requiredText),
```

Comment:
can it cause issues regarding males, because this question only appears when the contact is female